yarn-project/prover-client/src/orchestrator/orchestrator.ts

```diff
@@ -264,18 +294,26 @@ export class ProvingOrchestrator {
     logger.debug(`Enqueuing deferred proving for padding txs to enqueue ${txInputs.length} paddings`);
     this.deferredProving(
       provingState,
-      signal =>
-        this.prover.getEmptyPrivateKernelProof(
-          {
-            // Chain id and version should not change even if the proving state does, so it's safe to use them for the padding tx
-            // which gets cached across multiple runs of the orchestrator with different proving states. If they were to change,
-            // we'd have to clear out the paddingTx here and regenerate it when they do.
-            chainId: unprovenPaddingTx.data.constants.txContext.chainId,
-            version: unprovenPaddingTx.data.constants.txContext.version,
-            header: unprovenPaddingTx.data.constants.historicalHeader,
-          },
-          signal,
-        ),
+      wrapCallbackInSpan(
+        this.tracer,
+        'ProvingOrchestrator.prover.getEmptyPrivateKernelProof',
+        {
+          [Attributes.PROTOCOL_CIRCUIT_TYPE]: 'server',
+          [Attributes.PROTOCOL_CIRCUIT_NAME]: 'private-kernel-empty' as CircuitName,
```

Comment:
Ouch, why the cast?

Reply:
Oh, it captures typos `'private-kernel-empt' as CircuitName` fails to compile (missing `y` at the end)